grafana/resource_library_panel_test.go

```diff
@@ -0,0 +1,152 @@
+package grafana
+
+import (
+	"fmt"
+	"testing"
+
+	gapi "github.com/grafana/grafana-api-golang-client"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccLibraryPanel_basic(t *testing.T) {
+	CheckOSSTestsEnabled(t)
```

Comment:
You can merge in master and add the following line to ignore v7 tests
```suggestion
	CheckOSSTestsEnabled(t)
	CheckOSSTestsSemver(t, ">=8.0.0")
```